clang/test/CodeGen/X86/avx-cxx-record.cpp

```diff
@@ -11,6 +12,7 @@ struct XMM2 : XMM1<0>, XMM1<1> {
 };
 
 // CHECK: define{{.*}} @_Z3foov({{.*}} [[ARG:%.*]]){{.*}}
+// CLANG-20: define{{.*}} <4 x double> @_Z3foov()
```

Comment:
I think this is the only test which is different between clang 20 and clang 21, hence the change.